invokeai/frontend/web/src/features/nodes/components/flow/nodes/Invocation/fields/InputField.tsx

```diff
@@ -87,17 +97,20 @@ const InputField = ({ nodeId, fieldName }: Props) => {
   return (
     <InputFieldWrapper shouldDim={shouldDim}>
       <FormControl isInvalid={isMissingInput} isDisabled={isConnected} orientation="vertical" px={2}>
-        <Flex flexDir="column" w="full" gap={1}>
+        <Flex flexDir="column" w="full" gap={1} onMouseEnter={handleMouseOver} onMouseLeave={handleMouseOut}>
           <FieldContextMenu nodeId={nodeId} fieldName={fieldName} kind="input">
```

Comment:
I think we can safely remove this context menu. It's only purpose is to let you add and remove the fields and this PR handles that functionality in a much nicer way. Can delete the whole component.